v2/src/log_errno_linux.c

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include <errno.h>
+#include <string.h>
+
+#include "c_logging/log_errno.h"
+
+#define MESSAGE_BUFFER_SIZE 512
+
+static const char strerror_r_failure_message[] = "failure in strerror_r";
+
+// This array is not used for anything, but rather just to emit a compiler error
+static const char test_message_to_trigger_a_compiler_error[1 * (MESSAGE_BUFFER_SIZE >= sizeof(strerror_r_failure_message))] = { 0 }; /*this construct will generate a compile time error (array of size 0) when LOG_SIZE_REGULAR is not enough to hold even the failure message*/
```

Comment:
>LOG_SIZE_REGULAR [](http://example.com/codeflow?start=208&length=16)

Is this a wrong name?  Should this be MESSAGE_BUFFER_SIZE instead of LOG_SIZE_REGULAR

Reply:
Yes, fixed, ty